### PR TITLE
chore(docker): containerize Django application

### LINKED ISSUE #4: [Docker] Containerize the Django app

### LINKED ISSUE #36: [Nginx] Impement nginx w/ certibot
To access https://65.21.241.92:8042/, it will require an nginx instance running with the proper certifications

### LINKED ISSUE #39: [CONTRIBUTING] Update CONTRIBUTING.md file

### PR DESCRIPTION
**Pull request by luabida**
fix #4 
fix #36
fix #39

### CODE REVIEW

**Comment by luabida** [comment]
@fccoelho the only thing left behind in this PR will be the production certification blockage, the PR is ready to review

**Review by fccoelho — APPROVED** [review]

File: containers/nginx/nginx.conf [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    proxy_buffer_size   256k;
+
+    ssl_prefer_server_ciphers   on;
+    ssl_session_cache           shared:SSL:10m;
+    ssl_session_timeout         10m;
+    ssl_protocols               TLSv1 TLSv1.1 TLSv1.2;
+    ssl_ciphers                 EDH+CAMELLIA:EDH+aRSA:EECDH+aRSA+AESGCM:EECDH+aRSA+SHA256:EECDH:+CAMELLIA128:+AES128:+SSLv3:!aNULL:!eNULL:!LOW:!3DES:!MD5:!EXP:!PSK:!DDS:!RC4:!SEED:!IDEA:!ECDSA:kEDH:CAMELLIA128-SHA:AES128-SHA;
+
+    location / {
+      proxy_pass http://django:8042;
```
Comment (fccoelho) [inline containers/nginx/nginx.conf:40]: in compose-prod.yaml the port is **18042**, not 8042 as it is here. Maybe this is behind the problem in the deployment?

Comment (luabida) [reply containers/nginx/nginx.conf:40]: The 18042 is representing the http port 80, inside the mosqlimate docker network, the django port is exposed on 8042, should these two ports (nginx and django) match each other?

Comment (luabida) [reply containers/nginx/nginx.conf:40]:
```sh
docker exec -it mosqlimate_nginx_1 bash

root@nginx:/# wget http://django:8042

--2023-07-21 19:19:20--  http://django:8042/
Resolving django (django)... 192.168.240.3
Connecting to django (django)|192.168.240.3|:8042... connected.
HTTP request sent, awaiting response... 200 OK
Length: 2751 (2.7K) [text/html]
Saving to: 'index.html'

index.html                          100%[================================================================>]   2.69K  --.-KB/s    in 0s

2023-07-21 19:19:20 (75.5 MB/s) - 'index.html' saved [2751/2751]

root@nginx:/# cat index.html

<!DOCTYPE html>

<html>

  <head>
    <script src="/static/js/external/jquery.js"></script>

    <title>Home</title>
    <link rel="stylesheet" href="/static/css/main/base.css">
    <link rel="shortcut icon" href="/static/img/favicon.ico" type="image/x-icon">
    <link href="/static/fontawesomefree/css/fontawesome.css" rel="stylesheet" type="text/css">
    <link href="/static/fontawesomefree/css/brands.css" rel="stylesheet" type="text/css">
    <link href="/static/fontawesomefree/css/solid.css" rel="stylesheet" type="text/css">
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">


  </head>

  <body>
    <nav class="navbar navbar-expand-lg navbar-light bg-light font-golos">
      <div class="container-fluid">
        <!-- Navbar screen toggle -->
        <a class="navbar-brand" href="/">Mosqlimate</a>
        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
          <span class="navbar-toggler-icon"></span>
        </button>
        <!-- Navbar Items -->
        <div class="collapse navbar-collapse" id="navbarSupportedContent">
          <ul class="navbar-nav me-auto mb-2 mb-lg-0">
            <li class="nav-item">
              <a class="nav-link" aria-current="page" href="/">Home</a>
            </li>
            <li class="nav-item">
              <a class="nav-link" aria-current="page" href="/about/">About</a>
            </li>
            <li class="nav-item">
              <a class="nav-link" aria-current="page" href="/docs/">Documentation</a>
            </li>
            <li class="nav-item">
              <a class="nav-link" aria-current="page" href="/api/docs">API Demo</a>
            </li>
          </ul>
        </div>
        <!-- Navbar User & Login/out -->
        <div class="collapse navbar-collapse" id="navbarSupportedContent">
          <ul class="navbar-nav ms-auto mb-lg-0">
            <li class="nav-item">

            <li class="nav-item">
              <form method="post" action="/accounts/github/login/">
                <input type="hidden" name="csrfmiddlewaretoken" value="lABHbUKn947bsVrEZvWQEKe2HLHtbjMYSHeY8LXbBNG0iPaEYRUQR8iJf1mi9p5Y">
                <button type="submit" class="btn btn-primary me-auto" role="button">Login with Github</i></button>
              </form>

            </li>
          </ul>
        </div>
      </div>
    </nav>
    <!-- Page content -->
    <body>

<h1>Home</h1>

    </body>
</html>

```

**Pull request merged**